news/4.33/pde.html

```diff
@@ -46,6 +46,21 @@ <h2>Plug-in Development Environment</h2>
   <tr>
     <td id="editors" class="section" colspan="2"><h2>Editors</h2></td>
   </tr>
+  <tr id="osgirepositories">
+    <!-- https://github.com/eclipse-pde/eclipse.pde/pull/1285 -->
+    <td class="title">New Modify button in Execution Environment section</td>
+    <td class="content">
+      <p>
+      	New 'Modify..' button is added in the Execution Environment section of the Overview tab of the Manifest Editor.
+      </p>
+    <p>
+      <img src="images/EESection_Manifest.png" />
+    </p>
+  	  <p>
+  	  A new 'Modify...' button allows you to select a new EE, remove the old one, and update the classpath in one go. The Modify button is enabled only when a single EE is present in the EE section to modify.
+  	  </p>
```

Comment:
As far as I can tell, in the UI the `.classpath` file is not mentioned. The Context menu entry and preference is labeled `Java Build Path`. To make it easier for users not so deeply involved into Eclipse I think we should reuse that name. With that we would also cover the adjustments of the JDT settings.
```suggestion
      <p>
        The new 'Modify...' button allows you to select a new environment to replace the existing one and update the <em>Java Build Path</em> configuration in one action.
        The <code>Modify</code> button is enabled only when a single environment is present in this section.
      </p>
```